packages/app/pages/index.tsx

```diff
@@ -1,29 +1,59 @@
-import { User } from '@koh/common'
 import Router from 'next/router'
 import { ReactElement } from 'react'
-import { useHomePageRedirect } from '../hooks/useHomePageRedirect'
-import { useProfile } from '../hooks/useProfile'
-import { Spin } from 'antd'
+import { Button, Image, List } from 'antd'
+import styled from 'styled-components'
+
+const Container = styled.div`
+  margin-left: auto;
+  margin-right: auto;
+  text-align: center;
+  width: 500px;
+  padding-top: 100px;
+`
 
 export default function Home(): ReactElement {
-  const profile: User = useProfile()
-  const didRedirect = useHomePageRedirect()
-  if (profile && !didRedirect) {
-    Router.push('/courses')
-  } else {
-    return (
+  const login = () => {
+    Router.push('/login')
+  }
+
+  return (
+    <Container>
       <div
         style={{
-          position: 'absolute',
-          left: '50%',
-          top: '50%',
-          transform: 'translate(-50%, -50%)',
+          display: 'flex',
+          alignItems: 'center',
+          justifyContent: 'center',
         }}
       >
-        <Spin tip="Loading..." size="large" />
+        <img
+          src="/ubc_logo.png"
```

Comment:
We should ask about logo, i feel like we should use logo of helpme instead of just UBC